db/seeds.rb

```diff
@@ -0,0 +1,18 @@
+require 'active_record'
+require 'roo'
+require_relative '../app/models/verse'
+
+Load 100 verses from an .xlsx document into the Verses table
+xlsx_path = 'db/seed_data/100_verses.xlsx'
+xlsx = Roo::Spreadsheet.open(xlsx_path)
+sheet = xlsx.sheet(0)
+
+(2..101).each do |i|
+  notation = sheet.cell(i, 1)
+  verse_text = sheet.cell(i, 2)
+  Verse.create(scripture: verse_text, notation: notation, version: 'CSB', day: i - 1)
+end
+
+User.create(email: "isaac.tullos@gmail.com", first_name: "Isaac", last_name: "Tullos")
```

Comment:
Won't this raise an error if it's run more than once? We might want to split seeding a User out from the Verses, and they can be run separately or together. One approach is to create classes that you can call from a rake task:
```ruby
  # rake db:seed:users
  namespace :db do
    namespace :seed do
      desc "Seed Users from /db/seeds/users.yml"
      task :users => :environment do
        SeedUsers.run
      end
    end
  end
```

Then you have the code for that in the /lib/tasks/dv_seed_users.rake file:
```ruby
class SeedUsers
  def run
    er.create(email: "isaac.tullos@gmail.com", first_name: "Isaac", last_name: "Tullos")
  end 
```

Rake tasks can call other tasks, so you can have something that seeds everything, or just specific models.
https://www.blandersoft.com/short/rake-yaml-rails-seed/

Reply:
I actually meant to ask a question about this, so good looking out! I had just been commenting out whatever I didn't want to run, but splitting it into different tasks makes so much more sense. I'll go ahead and fix this before I merge!